test/cp.js

```diff
@@ -451,6 +451,45 @@ test('-R implies -P', t => {
   });
 });
 
+test('-Ru respects the -u flag recursively (don\'t update newer file)', t => {
+  const dir = `${t.context.tmp}/cp-Ru`;
+  const sourceDir = `${dir}/old`;
+  const sourceFile = `${sourceDir}/file`;
+  const destDir = `${dir}/new`;
+  const destFile = `${destDir}/file`;
+  [sourceDir, destDir].forEach(d => shell.mkdir('-p', d));
+  shell.ShellString('foo\n').to(sourceFile);
```

Comment:
nit: file contents could be more descriptive. How about `source file contents` and `destination file contents`?